src/CSS/Missions.module.css

```diff
@@ -0,0 +1,92 @@
+p {
+  margin: 0;
+}
+
```

Comment:
- [ ] Please don't add style HTML elements, use classes in place of elements to style, because they are globally available. 

 - ```p``` is globally available so it disturbing ```p``` element of rockets component.
 
![Screenshot 2023-06-28 200506](https://github.com/ankitt26/Space-Travelers-Hub/assets/108623895/60e17b95-83a2-4787-af07-5f68efec972b)

Reply:
use class like  ```.para { margin: 0; }```  in Missions.module.css.